master/static/srv/check_ready_logs.py

```diff
@@ -16,13 +16,22 @@
 
 BACKOFF_SECONDS = 5
 
+# backoff module provides function decorators that provide an easy to use 
+# backoff and retry mechanism. The on_exception decorator is used to perform 
+# retries after a certain amount of backoff time determined by the arguments
+# passed. The current invocation specifies that the retries should happen at a
+# constant backoff time (generated by the the built-in generator 
+# backoff.constant), for RequestException error, until a 500 error is returned.
+# NB: on_exception methods adds a jitter value by default to avoid collisions.
+
 # Since the service is virtually inaccessible by the user unless
 # the call completes, we may as well try forever or just wait for
 # them to kill us.
 @backoff.on_exception(  # type: ignore
-    lambda: backoff.constant(interval=BACKOFF_SECONDS),
+    backoff.constant,
     RequestException,
     giveup=lambda e: e.response is not None and e.response.status_code < 500,
+    interval=BACKOFF_SECONDS,
```

Comment:
i think we should also set max_tries.

Reply:
and we need to pin the version in the setup.py file still so this doesn't happen again if they make breaking changes

Reply:
Added max_tries and added requirements file to pin the `backoff` package version to `1.10.0`. Additionally, set up the CircleCI job `test-unit-master-python` inside the `test-unit` workflow to test the new python test.

Reply:
when i install i get >2, why pin to 1.10.0?

Reply:
I had 1.10.0 installed on my system. I didn't realize there was version 2.x. I will test with 2.x and update the PR.

Reply:
pinned backoff to the latest version `2.2.1`